salt/modules/zabbix.py

```diff
@@ -846,8 +1070,8 @@ def host_create(host, groups, interfaces, **connection_args):
     .. code-block:: bash
 
         salt '*' zabbix.host_create technicalname 4
-        interfaces='{type: 1, main: 1, useip: 1, ip: "192.168.3.1", dns: "", port: 10050}'
-        visible_name='Host Visible Name' inventory_mode=0 inventory='{"alias": "something"}'
+        interfaces='{if_type: 1, main: 1, useip: 1, ip_: "192.168.3.1", dns: "", port: 10050}'
```

Comment:
What is the gain from the change in user facing syntax? This line also clobbers an example of setting inventory information.

Reply:
Sorry, my mistake. Will fix it in next commit.

Reply:
The Zabbix API uses the following for interfaces in host.create:
```
"interfaces": [
            {
                "type": 1,
                "main": 1,
                "useip": 1,
                "ip": "192.168.3.1",
                "dns": "",
                "port": "10050"
            }
        ]
```

Why the change to if_type and ip_? This will affect user configurations and existing states, so is the rename necessary?

Reply:
You are right, it was a typo. Please, see my last [commit](https://github.com/saltstack/salt/pull/47398/commits/612f2a5f207db90254ff6259e01356d017ab62e3#diff-48649b036902f9e3cd75a774e29e1b91R1072) where it is fixed.